reward/reward_distributor_test.go

```diff
@@ -36,6 +125,26 @@ func newTestBalanceAdder() *testBalanceAdder {
 	return balanceAdder
 }
 
+func getTestConfig() *params.ChainConfig {
+	return &params.ChainConfig{
+		MagmaCompatibleBlock: big.NewInt(0),
+		KoreCompatibleBlock:  big.NewInt(0),
+		UnitPrice:            1,
+		Governance: &params.GovernanceConfig{
+			Reward: &params.RewardConfig{
+				MintingAmount: minted,
+				Ratio:         "34/54/12",
```

Comment:
If it's not too hassle, can you use `50/40/10` for a more realistic setting?

Reply:
Not sure if it can be done by this week, maybe let's fix this in future PR